app/graph/graphql_crud_operations.rb

```diff
@@ -337,76 +331,64 @@ def self.define_default_type(&block)
     end
   end
 
-  def self.field_published
-    proc do |_classname|
-      field :published do
-        type types.String
-
-        resolve ->(obj, _args, _ctx) { obj.created_at.to_i.to_s }
-      end
-    end
-  end
-
   def self.field_annotations
     proc do |_classname|
-      connection :annotations, -> { AnnotationUnion.connection_type } do
-        argument :annotation_type, !types.String
+      connection :annotations, -> { AnnotationUnion.connection_type }, 'Annotations describing this item' do
+        argument :annotation_type, !types.String, 'Comma-separated list of annotation types' # TODO Convert to array of strings
 
-        resolve ->(obj, args, _ctx) { obj.get_annotations(args['annotation_type'].split(',').map(&:strip)) }
+        resolve ->(obj, args, _ctx) {
+          obj.get_annotations(args['annotation_type'].split(',').map(&:strip))
+        }
       end
-    end
-  end
 
-  def self.field_annotations_count
-    proc do |_classname|
-      field :annotations_count do
-        type types.Int
-        argument :annotation_type, !types.String
+      field :annotations_count, types.Int, 'Count of annotations describing this item' do
+        argument :annotation_type, !types.String, 'Comma-separated list of annotation types' # TODO Convert to array of strings
 
         resolve ->(obj, args, _ctx) {
           obj.get_annotations(args['annotation_type'].split(',').map(&:strip)).count
         }
       end
+
+      field :annotation, -> { AnnotationType }, 'Latest annotation of a given type' do
+        argument :annotation_type, !types.String, 'Annotation type'
+
+        resolve ->(project_media, args, _ctx) {
+          project_media.get_dynamic_annotation(args['annotation_type'])
```

Comment:
We should be able to return any annotation here, not only dynamic (I think it does already, but just being sure)